spec/components/summary_card/component_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
```

Comment:
Just wondering about the organisation of this file and the preview component spec above. Currently this is:

```
spec/components/previews/summary_card_component_preview.rb
spec/components/summary_card/component_spec.rb
```

Considering we just went through the trouble of moving all the `summary_card` components into a single directory (`app/components

Reply:
Specs and views are now in the same folder and match what is in `app/components`

```
spec/components/summary_card/view_preview.rb
spec/components/summary_card/view_spec.rb
```

View component previews can also be found at `/view_components`